src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -496,7 +507,11 @@ public void display(int delay, DrawingPanel fishPanel)
 					if ((xf > posX - 40 && xf < posX + 40) && (yf > posY - 25 && yf < posY + 25)) {
 						info.add(new FishInfo(fishPanel.getWidth(), Math.random() * maxY, wf, hf));
 						if(health < maxHealth)
-							health++;
+							if (character_type){
+								health+=2;
```

Comment:
Spaces around your operators!

Reply:
fixed

Reply:
Is this fixed?  I don't see spaces in the PR...

Reply:
sorry, my bad, I have not pushed it.